### ISSUE TITLE
[Feature Request] Every SDK needs a DataDog sample for sending metrics and traces to DataDog agent

### ISSUE THREAD

**Issue by cretz** [temporalio/features#380]
### Describe the solution you'd like

We need to demonstrate SDK DataDog metrics/tracing integration with Temporal in a more clear way. People are doing this today, but they are struggling or have to figure a lot out themselves. It is important that the metrics _and_ tracing be demonstrated from workers/clients using the DataDog agent.

* [ ] Go sample
* [ ] Java sample
* [ ] TypeScript sample
* [ ] Python sample
* [ ] .NET sample

**Comment by Quinn-With-Two-Ns** [comment]
For the metrics side, is that just showing how to integrate Datadog with Prometheus?

**Comment by cretz** [comment]
> For the metrics side, is that just showing how to integrate Datadog with Prometheus?

Exactly! I suspect the samples are more education than code/difficulty. I would support that education part being in docs instead of README of sample if/when we can.
